scripts/initializers/index.js

```diff
@@ -60,6 +60,7 @@ export default async function initializeDropins() {
   // Initialize Global Drop-ins
   await import('./auth.js');
   import('./cart.js');
+  await import('./search.js');
```

Comment:
Is search considered a global dropin? I don't think so.

It would be initialized on `search` routes, so within `blocks/header/searchbar.js` and `blocks/product-list-page`.

Reply:
I realize I said this and then noticed that cart has similar behavior (an "all page" widget as minicart, and a dedicated page at /cart) so I'm not so sure how we want to do this.